builder/remotecontext/git.go

```diff
@@ -21,9 +22,14 @@ func MakeGitContext(gitURL string) (builder.Source, error) {
 	}
 
 	defer func() {
-		// TODO: print errors?
-		c.Close()
-		os.RemoveAll(root)
+		err := c.Close()
+		if err != nil {
+			logrus.Errorf(err)
```

Comment:
These log entries need some extra context about what's going on otherwise the errors won't be very helpful in the logs.

You can use something like `logrus.WithField("action", "MakeGitContext").WithField("module", "builder").WithField("url", gitURL).WithError(err).Error("error while closing git context")`

Reply:
Ah! Thanks!